cmd/checkmetrics/basefile.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+	log "github.com/Sirupsen/logrus"
+)
+
+// metric is the slice of Metrics imported from the TOML config file
```

Comment:
Conventionally with go, the type is documented immediately above it, so I think this comment needs to be immediately above the `metrics` slice below.